brainrender_napari/widgets/atlas_manager_view.py

```diff
@@ -116,3 +133,74 @@ def get_tooltip_text(cls, atlas_name: str):
         else:
             raise ValueError("Tooltip text called with invalid atlas name.")
         return tooltip_text
+
+
+class AtlasManagerFilter(QWidget):
```

Comment:
Very nice, I would move this into its own file `widgets/atlas_manager_filter.py` though.